src/cluster.h

```diff
@@ -239,11 +240,38 @@ typedef struct {
     unsigned char bulk_data[3]; /* 3 bytes just as placeholder. */
 } clusterMsgModule;
 
+/* The cluster supports optional extension messages that can be sent
+ * along with ping/pong/meet messages to give additional info in a 
+ * consistent manner. */
+typedef enum {
+    CLUSTERMSG_EXT_TYPE_HOSTNAME,
+} clusterMsgPingtypes; 
+
+/* Helper function for making sure extensions are eight byte aligned. */
+#define EIGHT_BYTE_ALIGN(size) ((((size) / 8) + 1) * 8)
```

Comment:
```suggestion
#define EIGHT_BYTE_ALIGN(size) ((((size) + 7) / 8) * 8)
```

Currently rounding up when already aligned, I assume it's a mistake and not intentional?